emissionsapi/web.py

```diff
@@ -98,6 +99,63 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def cache_with_session(begin_name=None, end_name=None):
```

Comment:
Make it possible to define the name of the begin and end here makes it flexible, but also more complex. Let's just think about it one more time.